__tests__/requireUniversalModule.js

```diff
@@ -244,8 +245,7 @@ describe('requireAsync: requires module asynchronously on the client, returning
 
 describe('addModule: add moduleId and chunkName for SSR flushing', () => {
   it('babel', () => {
-    flushModuleIds() // insure sets are empty:
-    flushChunkNames()
+    clearChunks()
```

Comment:
Do you want more explicit tests for `clearChunks()`? I'm not sure if these fully exercise the function.

Reply:
na, it's good. I'm not too worried.